ros2topic/test/test_use_sim_time.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2022 Sony Corporation.
```

Comment:
Sorry I should have commented before, but correct name should be applied.

```suggestion
# Copyright 2022 Sony Group Corporation.
```

Reply:
updated.